db/migrate/20191101184930_create_vendors.rb

```diff
@@ -0,0 +1,10 @@
+class CreateVendors < ActiveRecord::Migration[5.1]
+  def change
+    create_table :vendors do |t|
+      t.string :email
```

Comment:
Like what @edwardyuendev said, this should've come with an ID and the various timestamp attributes.

Reply:
Should I add them manually then?